test/common/odp_cunit_common.h

```diff
@@ -80,7 +80,8 @@ int odp_cunit_run(void);
  * When 'priv' is 0, the same argument pointer (arg[0]) is passed to all threads. Otherwise,
  * a pointer is passed (from arg[]) to each thread. Returns 0 on success.
  */
-int odp_cunit_thread_create(int num, int func_ptr(void *arg), void *const arg[], int priv);
+int odp_cunit_thread_create(int num, int func_ptr(void *arg), void *const arg[],
+			    int priv, int sync);
```

Comment:
Something could be said about what `sync` controls.

Reply:
Sure, added comment in V2.